app/models/user.rb

```diff
@@ -528,7 +528,7 @@ def cas?
     end
 
     def machine?
-      minimal? || api? || created_by_provider? || open_id? || cas? || oauth2?
+      minimal? || api? || open_id? || cas? || oauth2?
```

Comment:
@HemaHG it needs a description, why this change is being made.. i don't get it. maybe add it to a PR description. thank you

Reply:
@HemaHG If this change is intended, it requires a test because it is not obvious at all.